### ISSUE TITLE
Training Labels Might Be Not Correct?

### ISSUE THREAD

**Issue by mylife126** [YeongHyeon/Super-Resolution_CNN-PyTorch#2]
Dear Author,
when you train the network, in the loss function part, you assigned the "label" to be **X_tr_t** instead of **Y_tr_t**. Should the label be the actual High Resolution Picture? Thanks!  

`           
            while(True):
            X_tr, Y_tr, X_tr_t, Y_tr_t, terminator = dataset.next_train(batch_size=batch_size)

            img_recon = neuralnet.model(X_tr_t.to(neuralnet.device))
            mse = neuralnet.mse(input=X_tr_t.to(neuralnet.device), target=img_recon.to(neuralnet.device))`

**Comment by YeongHyeon** [comment]
Dear 'mylife126',
I have updated that error now.
I think that was a missing part of the upload process after writing the code.
Thank you for informing me!
